core/src/test/java/org/elasticsearch/search/aggregations/AggregationsBinaryIT.java

```diff
@@ -41,6 +42,8 @@
 import static org.hamcrest.core.IsNull.notNullValue;
 
 @ESIntegTestCase.SuiteScopeTestCase
+@AwaitsFix(bugUrl = "needs fixing after the search request refactor. Do we need agg binary?")
```

Comment:
I didn't know they were removed...when did that happen? I still see occurrences of them in the codebase. Maybe the removal left done some leftover?